lib/auth0/api/authentication_endpoints.rb

```diff
@@ -150,9 +150,10 @@ def signup(email, password, connection_name = UP_AUTH)
       # @see https://auth0.com/docs/api/authentication#change-password
       # @see https://auth0.com/docs/connections/database/password-change
       # @param email [string] User's current email
-      # @param password [string] User's new password; empty to trigger a
-      #   password reset email
+      # @param password [string] User's new password. This is only available
+      #   on legacy tenants with the change_pwd_flow_v1 flag enabled
```

Comment:
Honestly not sure, but do we expose the flag identifier anywhere in a customer-facing fashion? If not, it might be better to identify this as the "Change Password Flow" toggle indicated in their tenant account settings, under the advanced tab: if v2 is enabled it's locked out for them.